flake.nix

```
@@ -48,7 +48,14 @@
         forAllSystems = nixpkgs.lib.genAttrs nixpkgs.lib.systems.flakeExposed;
       in
       {
-        formatter = forAllSystems (system: nixpkgs.legacyPackages.${system}.nixfmt-tree);
+        formatter = forAllSystems (
+          system:
+          nixpkgs.legacyPackages.${system}.nixfmt-tree.override ({
+            settings = {
+              tree-root-file = "release.json";
+            };
+          })
+        );
```

```suggestion
        formatter = forAllSystems (
          system:
          nixpkgs.legacyPackages.${system}.nixfmt-tree.override {
            settings = {
              tree-root-file = "release.json";
            };
          }
        );
```

Just minor nit for redundant parens